pkg/deploy/util/util.go

```diff
@@ -228,6 +234,19 @@ func EncodeDeploymentConfig(config *deployapi.DeploymentConfig, codec runtime.Co
 	return string(bytes[:]), nil
 }
 
+func NewControllerRef(config *deployapi.DeploymentConfig) *metav1.OwnerReference {
+	blockOwnerDeletion := true
```

Comment:
@tnozicka I saw BoolPtr() in some vendor package (not kube). Since we are doing this in a lot of places, it sounds reasonable to me:

```
func BoolPtr(r bool) *bool {
  return &r
}

func BoolTruePtr() *bool {
  return BoolPtr(true)
}
```